datafusion/row/src/accessor.rs

```diff
@@ -116,9 +116,9 @@ macro_rules! fn_get_idx_scalar {
 
 impl<'a> RowAccessor<'a> {
     /// new
-    pub fn new(schema: &Schema, row_type: RowType) -> Self {
```

Comment:
Thanks for the PR. This is always great to cut out the unused code. For public functions though it can be a user interface change for downstream projects btw?

Reply:
Thanks for bringing this out! I've created an issue, added an API change label, and edited the PR descriptions accordingly.